cluster/manifests/zmon-aws-agent/deployment.yaml

```diff
@@ -5,7 +5,7 @@ metadata:
   namespace: "visibility"
   labels:
     application: "zmon-aws-agent"
-    version: "v77-12-g3bf9a28-zv156"
+    version: "v77-14-g9daa9d1-zv156
```

Comment:
string looks wrong with missing `"` in the end?

Reply:
Oops :( sorry

Reply:
Fixed!